react/ui-components/src/atoms/TableCell.js

```
@@ -131,14 +152,44 @@ const TableCell = ({
+        return (
+          <MultiSelectDropdown
+            {...cellData}
+            onSelect={(newValue) => {
+              updateTableData(rowIndex, cellIndex, newValue);
+              if (cellData?.onSelect) {
+                cellData.onSelect(newValue);
+              }
+            }}
+          ></MultiSelectDropdown>
+        );
       case "custom":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)